sys-auth/AusweisApp2/AusweisApp2-1.20.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Official authentication app for German ID cards and residence permits"
+HOMEPAGE="https://www.ausweisapp.bund.de/"
+SRC_URI="https://github.com/Governikus/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EUPL-1.2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwebsockets:5[qml]
+	dev-qt/qtquickcontrols2:5
```

Comment:
I don't see this depdency used? It that linked with your local build?

Reply:
-- BUILD_SHARED_LIBS: OFF
CMake Error at cmake/Libraries.cmake:19 (FIND_PACKAGE):
By not providing "FindQt5QuickControls2.cmake" in CMAKE_MODULE_PATH this
project has asked CMake to find a package configuration file provided by
"Qt5QuickControls2", but CMake did not find one.

Could not find a package configuration file provided by "Qt5QuickControls2"
(requested version 5.10) with any of the following names:

Qt5QuickControls2Config.cmake
qt5quickcontrols2-config.cmake

Reply:
According to that error, better move it to DEPEND?

Reply:
Now moved to DEPEND